src/wasm/types.rs

```diff
@@ -75,6 +75,7 @@ type EthSignature = {
   s: Uint8Array;
   v: Uint8Array;
 };
+type EthAddressCase = 'Upper' | 'Lower' | 'Mixed';
```

Comment:
it's unexpected that these are uppercase and not lowercase. How come? The `EthAddressCase` struct is marked to camelCase the names


https://github.com/BitBoxSwiss/bitbox-api-rs/blob/6dd99fd762aa342f7f89e738dac7da5281df8658/src/shiftcrypto.bitbox02.rs#L1854

```
#[cfg_attr(feature = "wasm", serde(rename_all = "camelCase"))]
```